File: snippets/fsharp/VS_Snippets_CLR_System/system.char.toupper/fs/toupper5.fs
// <Snippet2>
open System
open System.Globalization

let cultures= 
    [ CultureInfo.CreateSpecificCulture "en-US"
      CultureInfo.InvariantCulture
      CultureInfo.CreateSpecificCulture "tr-TR" ]

let chars = [| 'ä'; 'e'; 'E'; 'i'; 'I' |]

printfn "Character     en-US     Invariant     tr-TR"
for ch in chars do
    printf $"    {ch}"
    for culture in cultures do
        printf $"{Char.ToUpper(ch, culture),12}"
    printfn ""


// The example displays the following output:
//       Character     en-US     Invariant     tr-TR
//           ä           Ä           Ä           Ä
//           e           E           E           E
//           E           E           E           E
//           i           I           I           İ
//           I           I           I           I
// </Snippet2>

File: snippets/csharp/System/Char/ToUpper/toupper5.cs
// <Snippet2>
using System;
using System.Globalization;

public class Example2
{
    public static void Run()
    {
        CultureInfo[] cultures = { CultureInfo.CreateSpecificCulture("en-US"),
                                CultureInfo.InvariantCulture,
                                CultureInfo.CreateSpecificCulture("tr-TR") };
        char[] chars = { 'ä', 'e', 'E', 'i', 'I' };

        Console.WriteLine("Character     en-US     Invariant     tr-TR");
        foreach (char ch in chars)
        {
            Console.Write($"    {ch}");
            foreach (var culture in cultures)
                Console.Write($"{char.ToUpper(ch, culture),12}");

            Console.WriteLine();
        }
    }
}
// The example displays the following output:
//       Character     en-US     Invariant     tr-TR
//           ä           Ä           Ä           Ä
//           e           E           E           E
//           E           E           E           E
//           i           I           I           İ
//           I           I           I           I
// </Snippet2>
